<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>timeControlClass</class>
 <widget class="QMainWindow" name="timeControlClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>timeControl</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="4">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QTimeEdit" name="timeEditLimit">
      <property name="currentSection">
       <enum>QDateTimeEdit::HourSection</enum>
      </property>
      <property name="time">
       <time>
        <hour>4</hour>
        <minute>0</minute>
        <second>0</second>
       </time>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QCheckBox" name="checkBoxOnOffTimer">
      <property name="text">
       <string>Таймер вкл/выкл</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="buttonSetLimit">
      <property name="text">
       <string>установить ограничение</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QPushButton" name="buttonRemove5Minutes">
      <property name="text">
       <string>-5 минут</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="8">
     <widget class="QLabel" name="labelTimerView">
      <property name="font">
       <font>
        <pointsize>72</pointsize>
       </font>
      </property>
      <property name="text">
       <string>00:00:00</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QPushButton" name="buttonAdd5Minutes">
      <property name="text">
       <string>+5 минут</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="timeControl.qrc"/>
 </resources>
 <connections/>
</ui>
